src/mix/api/channels.ts

```diff
@@ -30,3 +30,25 @@ export async function renameChannel(client: MixClient, requestParams: ChannelsRe
     data: body,
   })
 }
+
+/**
+ * Create a new channel in a project.
+ *
+ * @category channels
+ */
+export async function createChannel(client: MixClient, requestParams: ChannelsCreateParams): Promise<MixResponse> {
+  debug('createChannel()')
+  const {projectId, displayName, modes = ['MODE_UNSPECIFIED'], color = 'COLOR_UNSPECIFIED'} = requestParams
```

Comment:
If create behaves like configure, the backend won't like MODE_UNSPECIFIED and COLOR_UNSPECIFIED. Something to verify and this means users may need to be forced enter a valid value for both.